src/caselawclient/client_helpers/search_helpers.py

```diff
@@ -14,8 +16,11 @@ def search_judgments_and_parse_response(
 
     :return: The parsed search response as a SearchResponse object
     """
-    return SearchResponse.from_response_string(
-        api_client.search_judgments_and_decode_response(search_parameters)
+    return SearchResponse(
```

Comment:
Hmm, I feel like this could go further - could the the `Client` have  one public `search` method which takes `search_parameters`, and returns a `SearchResult`, handling the `search_judgments_and_decode_response` logic, parsing the element tree, and injecting itself as a dependency into the result? That way `search_helpers.py` would not need to know or care about the internals.

Reply:
This has been added to the list of further refactoring.